### DISCUSSION TITLE
How do I specify a BoolFlag with the inverse value set?

### DISCUSSION THREAD

**Discussion by joebonneau** [urfave/cli#1803] [Q&A]
Hello,

I see in the source code that there is a module called [flag_bool_with_inverse.go](https://github.com/urfave/cli/blob/main/flag_bool_with_inverse.go), but I can't seem to find any documentation of how to implement this. If I just happened to miss it in the docs, sorry in advance.

Thanks,
Joe

**Comment by dearchap** [comment]
@joebonneau Yes you can use the BoolWithInverseFlag similar to how the BoolFlag is used and it will automatically setup 2 flags internally for the xx and the no-xx flags names.

**Reply by joebonneau** [reply]
Thanks for the answer, @dearchap. When I try to use `cli.BoolWithInverseFlag`, I get a compiler error saying that it's undefined. I'm using v2.25.7.

Seems like `BoolWithInverseFlag` might only be available in 3.0.0alpha?

**Reply by dearchap** [reply]
Ah yes it is only available in v3.
